wiki/Legal/Copyright/ru.md

```diff
@@ -1,22 +1,20 @@
 ---
 legal: true
-no_native_review: true
-needs_cleanup: true  # нужно сравнить с оригиналом и добавить дату последнего перевода
 ---
 
 # Политика osu! в отношении авторского права
 
-osu! является игрой в значительной степени ориентированной на сообщество. Все игровые уровни (битмапы) создаются и загружаются участниками сообщества, чаще всего без получения разрешения от соответствующих правообладателей. Мы стараемся не рекламировать или получать прибыль непосредственно от загруженного контента, а игра полностью финансируется за счет пожертвований пользователей.
+osu! — игра, которая сильно ориентирована на сообщество. Все игровые уровни (карты) создаются и загружаются участниками сообщества, чаще всего — без получения разрешения от соответствующих правообладателей. Мы стараемся не рекламировать загруженный контент и не извлекать прибыль из него непосредственно, а игра финансируется полностью за счет пожертвований пользователей.
 
-Большая часть полученых пожертвований, которые мы получаем, используются для [лицензирования музыки](/wiki/Legal/Music_licensing). Запросы для лицензирования музыки могут быть отправлены на [music@ppy.sh](mailto:music@ppy.sh).
+Большая часть получаемых нами пожертвований вкладывается в [лицензирование музыки](/wiki/Legal/Music_licensing). Обращения, связанные с лицензированием, могут быть отправлены на [music@ppy.sh](mailto:music@ppy.sh).
 
-В противном случае, osu! приняли следующую политику в отношении нарушения авторских прав в соответствии с Законом об авторском праве в цифровую эпоху, глава 17 статья 512 кодекса США.
+Во всех остальных случаях osu! использует следующую политику в отношении нарушения авторских прав, в соответствии с Законом об авторском праве в цифровую эпоху, глава 17 статья 512 кодекса США.
 
-Контактные данные представителя osu!, уполномоченного получать жалобы о нарушении авторских прав (далее — Представитель), указаны в конце данного документа и хранятся в архиве Бюро авторского права США (<https://www.copyright.gov/dmca-directory/>). osu! серьёзно относится к защите авторских и других прав интеллектуальной собственности.
+Контактные данные представителя osu!, уполномоченного получать жалобы о нарушении авторских прав (далее — Представитель), указаны в конце данного документа и хранятся в архиве Бюро авторского права США ([https://www.copyright.gov/dmca-directory/](https://www.copyright.gov/dmca-directory/)). osu! серьёзно относится к защите авторских и других прав интеллектуальной собственности.
```

Comment:
this was correct before

Reply:
I know, but it's taken from the original page, so I assumed there had to be some meaning in doing otherwise

Reply:
ah well i guess it can be cleaned up in the original though